ClosedXML/Excel/Drawings/XLPictures.cs

```diff
@@ -74,7 +74,7 @@ public IXLPicture Add(Bitmap bitmap, string name)
 
         public IXLPicture Add(string imageFile)
         {
-            using (var fs = File.Open(imageFile, FileMode.Open))
+            using (var fs = File.OpenRead(imageFile))
```

Comment:
I thought `File.Open(imageFile, FileMode.Open)` and `File.OpenRead(imageFile)` were equivalent. What's the diffierence?

Reply:
![image](https://user-images.githubusercontent.com/19576939/98463546-438d0c80-21d5-11eb-9093-0222d40658be.png)

Reply:
Adding to it, Default [open ](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.open?view=net-5.0)- opens `FileStream` on specified path with read/write access with no sharing. I am seeing failures in our Asp.Net Core app because of that.